ospo-book/chapters/03-chapter.md

```diff
@@ -1 +1,146 @@
+* Which organization's processes have areas for improvement? 
+* How can open source support those improvements? 
+* How can you make workers champions for open source?
+* How can the message be effectively transmitted to management for their understanding?
+
+
+
+
+
+Moreover, it is important to [identify the open source projects and communities with which you will work](https://todogroup.org/guides/participating/). Developing a plan for how you will support these initiatives is crucial. Additionally, you will need to [consider the role that open source will play in your organization and how you will integrate open source solutions into your existing IT infrastructure](https://todogroup.org/guides/outbound-oss/).
+
+### Making Open Source Relevant outside the organization
+
+Organizations are subject to various external regulators that influence and shape their policies and processes. These regulators ensure compliance with legal requirements, ethical standards, and industry-specific guidelines. Some external regulators include:
+
+* Government Agencies: Government bodies establish and enforce laws and regulations that impact organizations.
+* Industry Regulators: Many industries have their own regulatory bodies or professional associations that set guidelines and standards for organizations to follow.
+* Consumer Protection Agencies: Consumer protection agencies ensure that organizations provide fair and safe products or services to consumers.
+
+In order for open source to be successful and sustainable within an organization, it is crucial to collaborate not only with the open source community but also with external regulators. This collaboration ensures a clear understanding of open source principles when creating policies that affect the ecosystem. The primary objective is to work together and make informed decisions by fully grasping the implications on open source and its community.
+
+![collaboration-outside](https://github.com/todogroup/ospology/assets/43671777/33b831ed-fbcd-42ad-bc62-a04d5467ddfc)
+
+
+Consequently, when formulating strategies, it is essential for the Open Source Program Office (OSPO) to develop a plan on how to approach and communicate with these regulators, clearly defining the roles they will play in the policy-making process.
+
+
+## Assessing Maturity of Open Source Program Office
+`✅ Assessment`
+
+Many times, the OSPO may not be able to perform all these tasks at once and will need to assess the previous steps to successfully add new milestones to their roadmap. 
+This is where [maturity models](https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/) come into play.
+
+The [OSPO maturity model](https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/) can help organizations to understand the progress they are making towards building a mature OSPO, and to identify the areas where they need to focus their efforts.
```

Comment:
The OSPO maturity model is very similar to the open source adoption maturity model from an organizational point of view. Maybe it would be better to describe it from this perspective and position the OSPO as enabler and facilitator of this organizational evolution towards higher open source competence, rather than developing in itself. The OSPO is also a means to an end to help the organization to address risks when using open source and realize the potential the open source model provides. There is not much value in having or evolving an OSPO just for the sake of having an OSPO.